examples/packages/flow-types/src/index.js

```diff
@@ -0,0 +1,56 @@
+// @flow strict-local
```

Comment:
What is this file and package for exactly? Is it to showcase how to apply flow types, or to test the types? If the latter, then tests should live in the package they're defined in (i.e. `@compiled/react`). You can do something similar to what I setup for the RLL repository, so that you have test files like so: https://github.com/atlassian-labs/react-loosely-lazy/blob/master/packages/core/react-loosely-lazy/src/lazy/__tests__/lazy.flow.test.js

Reply:
The main intention was to break builds if the type imports failed when importing the package. Moving to regular unit test files sounds good to me, I don't think there's a huge value to importing the package retrospectively - things are gonna break elsewhere first.

Reply:
- [ ] Move to test files in package